lms/templates/widgets/segment-io-footer.html

```diff
@@ -0,0 +1,14 @@
+% if settings.LMS_SEGMENT_KEY:
+    <!-- begin segment footer -->
```

Comment:
Thanks for catching this, @rlucioni. I naively thought that the purpose of main/base.html was to centralize such logic. It would be good to make this all more DRY, if we can. I wonder if we can even put it in common so that Studio and LMS share the same implementation too.